include/wx/generic/dragimgg.h

```diff
@@ -233,12 +233,11 @@ class WXDLLIMPEXP_CORE wxGenericDragImage: public wxObject
     bool            m_isDirty;
     bool            m_isShown;
     wxWindow*       m_window;
-    wxDC*           m_windowDC;
 
 #ifdef wxHAS_NATIVE_OVERLAY
```

Comment:
**For one hand**, I've updated `wxGenericDragImage` to use the new (`wxOverlay`) API to see how well (or bad!) it will behave with the changes:
Under **MSW**, `wxGenericDragImage` works perfectly fine (you can hardly distinguish between the native and the the generic version there).

Under **GTK3**, `wxGenericDragImage` works now correctly in fullscreen mode (at least under X11) which was broken before.

**For the other hand**, adapting `wxCaret` to use the generic version of `wxOverlay` was somewhat trivial, which is not the case for `wxGenericDragImage`, and i don't know what to do about it for the moment.

I'll post in the wx-dev about "why wxCaret should use wxOverlay, the generic version and not the native one".